Demos/DemoLightbox/Podfile

```diff
@@ -3,6 +3,9 @@ platform :ios, '8.0'
 use_frameworks!
 inhibit_all_warnings!
 
-pod 'Lightbox', path: '../../'
-pod 'Sugar'
-pod 'Hue', git: 'https://github.com/hyperoslo/Hue'
+target 'DemoLightbox' do 
+	pod 'Lightbox', path: '../../'
+	pod 'Sugar'
+	pod 'Hue', git: 'https://github.com/hyperoslo/Hue'
```

Comment:
@aashishdhawan yeah, you're right. I see some are also using Sugar for 1 or 2 functions